pylintrc

```diff
@@ -180,4 +459,4 @@ int-import-graph =
 [EXCEPTIONS]
 overgeneral-exceptions = Exception
 
-# a795e1dfab29b0fe16d892e7a0ee736ec2927408
+# a78cbfddf357eb444028233318f94b1c8c343447
```

Comment:
what is this ?

Reply:
This is auto generated file using the edx-lint command 
`edx_lint write pylintrc`